docs/knowledge_base/index.md

```diff
@@ -200,6 +200,19 @@ hide:
       </div> 
     </a>
   </li>
+  <li data-keywords="gui vizelement markdown">
+    <a class="tp-content-card tp-content-card--horizontal tp-content-card--small" href="tutorials/chatbot/">
+      <header class="tp-content-card-header">
+        <img class="tp-content-card-icon" src="tutorials/images/icon-code.svg">
+      </header>
+      <div class="tp-content-card-body">
+        <h4>LLM ChatBot</h4>
+        <p>
+          Create a chatbot website using Taipy and an LLM API. 
```

Comment:
I don't really like the website. Why did you change the description from the other one?

Reply:
Understood, I wanted to ensure that the uploading to Cloud part is included in the description. Any suggestions?

Reply:
Changed to "interface"